pom.xml

```diff
@@ -76,6 +76,7 @@
     <project.build.outputEncoding>UTF-8</project.build.outputEncoding>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.findbugs.failure.strict>true</maven.findbugs.failure.strict>
+    <maven-release-plugin.version>2.5.3</maven-release-plugin.version>
```

Comment:
I agree with @aheritier BTW. If a developer feels that version needs to be configurable, the approach is right